provider/management/commands/delete_expired_grant_tokens.py

```diff
@@ -0,0 +1,55 @@
+"""
+Command to delete expired OAuth2 grant tokens from the oauth2_grant table.
+
+Accepts a parameter to determine how many days back to consider a token expired.
+"""
+
+import logging
+
+from django.core.management.base import BaseCommand, CommandError
+from django.db import transaction
+
+from provider.oauth2.models import Grant
+
+log = logging.getLogger(__name__)
+
+class Command(BaseCommand):
+    """
+    Example usage: ./manage.py lms --settings=devstack delete_expired_grant_tokens.py --days_back 2
+    """
+    help = 'Deletes OAuth2 grant tokens expired for longer than days_back days.'
+
+    # Default value for days_back.
+    DEFAULT_DAYS_BACK = 1
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--days_back',
```

Comment:
As a heads up, for DOP this is true, in the DOT world, accesstokens are linked to refresh tokens, so you can't delete accesstokens until the refreshtoken expires.
Luckily they have their own management command (which of course nobody runs because nobody owns DOT).